packages/survey-creator-core/src/creator-base.ts

```diff
@@ -643,7 +643,8 @@ export class CreatorBase extends Base
    *- options.editorOptions  options that can be changed.
    *- options.editorOptions.allowAddRemoveItems a boolean property, true by default. Set it false to disable add/remove items in array properties. For example 'choices', 'columns', 'rows'.
    *- options.editorOptions.allowRemoveAllItems a boolean property, true by default. Set it false to disable remove all items in array properties. For example 'choices', 'columns', 'rows'.
-   *- options.editorOptions.showTextView a boolean property, true by default. Set it false to disable "Manual Entry" tab for "choices" property.
+   *- options.editorOptions.enabledTextView a boolean property, true by default. Set it false to disable "Manual Entry" tab for "choices" property.
+   *- options.editorOptions.showTextView a boolean property, true by default. Set it false to hide "Manual Entry" tab for "choices" property.
```

Comment:
*- options.editorOptions.showTextView a boolean property, `true` by default. Set it `false` to hide the "Edit"button for the "choices" property.